examples/simple-platform/user-walk-through.md

```diff
@@ -0,0 +1,107 @@
+# User Walk-through of the Platform Example
+
+## Create an account and complete the OAuth flow with GitHub
+
+1. Open the accounts app in your browser at `http://localhost:3025`.
+
+1. Open the browser's developer tools to the console tab and verify that you
+see the `interbit-middleware/CHAIN_BLOCKING` status logged to the console for
+the `accountsPrivate` chain.
+
+   The Interbit middleware prints a number of messages to the console so you
+can see, for instance, when chains have loaded and actions have been
+dispatched. The account app is fully up and running when you see the above
+message. At this point, you can dispatch actions to your
+private account chain.
```

Comment:
Is there one chain per Interbit account? If so, we should say `...to the private blockchain containing your account information`. If not, we should replace `account` with `accountsPrivate`.